.gitignore

```diff
@@ -14,3 +14,7 @@ pom.xml.releaseBackup
 .idea
 *.sublime-project
 *.sublime-workspace
+
+# vim
+*.swp
+Session.vim
```

Comment:
I have done a local rebase and split this commit, but I do not know how to push this back without getting into trouble. When I push the rebase, it is obviously rejected. When I pull, I get lots of conflicts. Perhaps I should start over on a different branch or delete this branch and resubmit the pull request?